### PR TITLE
Project page

### PR DESCRIPTION
**Pull request by luederb**

### CODE REVIEW

File: frontend/src/Header/Header.css [hunk 1]
```diff
@@ -1,12 +1,5 @@
 header {
-    grid-area: header;
-    background-color: var(--standardBackgroundColor);
-    color: var(--standardFondColor);
-    display: grid;
-    grid-template-columns: 4fr 1fr;
-    align-items: center;
-    border-radius: 5px 5px 0 0;
-    border-bottom: 3px solid var(--highlightColor);
+
```
Comment (lostspezi) [inline frontend/src/Header/Header.css]: check sonar isse, you could delete this here

**Review by lostspezi — CHANGES_REQUESTED** [review]
So far so good, just one little thing ;-)

File: frontend/src/Header/Header.css (hunk 1 above)
Comment (luederb) [reply frontend/src/Header/Header.css]: ahh, yes you're right. Thanks

**Review by lostspezi — APPROVED** [review]
LGTM!:)

**Review by MarcellNF — APPROVED** [review]
LGTM!:)

**Pull request merged**